data/transaction.go

```diff
@@ -52,7 +52,7 @@ func (t *Transaction) GetData() []byte {
 
 // GetRcvAddr will return transaction receiver address
 func (t *Transaction) GetRcvAddr() []byte {
```

Comment:
GetReceiverAddress ?

Reply:
I have to keep this name in order to implement the interface  `TransactionWithFeeHandler`